x509/src/lib.rs

```diff
@@ -16,14 +16,17 @@ extern crate std;
 
 mod attribute;
 mod certificate;
+pub mod extensions_utils;
 mod general_name;
+pub mod pkix_extensions;
+pub mod pkix_oids;
```

Comment:
Would it make sense to have `pub mod pkix` with separate `extensions` and `oids` submodules?

Reply:
Offhand, I don't think so. pkix_extensions should probably be named extensions (or cert_extensions with CRL extensions split off) if we are going to regroup. The RDN OIDs in the pkix_oids file are used indirectly by the certificate module too so there's affinity with the sibling certificate module too.